<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Gestion_agents">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>951</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>541</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
      <property name="title">
       <string>Gestion agents</string>
      </property>
      <widget class="QPushButton" name="pushButton_Chercher">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Search for agent</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QPushButton" name="print">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Print payment sheet</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditchercher">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>110</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tab_recherche">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>291</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="statistics">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>541</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Ajouter agent</string>
      </property>
      <widget class="QLabel" name="Ajout_cin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="admissiondate">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Admission date</string>
       </property>
      </widget>
      <widget class="QLabel" name="Ajout_nom">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="email">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>320</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberofabsences">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Number of absences</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCIN">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>99999999</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numberofabsences">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adress">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>200</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="comfirm">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>250</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Comfirm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="return_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QLabel" name="position">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
      <widget class="QLabel" name="salary">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Salary</string>
       </property>
      </widget>
      <widget class="QLabel" name="adress">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Adress</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Salary">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>280</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>320</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_position">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>240</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QTableView" name="tab_agents">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modify">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>260</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditsupprimer">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>310</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>350</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Admission Date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Name :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of absences :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>140</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Adress :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>170</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Position :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>200</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Salary :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>230</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditcinmodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditnamemodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditnumberofabsencesmodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditadressmodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditpositionmodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditsalarymodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditemailmodifier">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="triposition">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>410</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>tri position</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trinumberofabsences">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>tri number of absences</string>
      </property>
     </widget>
     <widget class="QPushButton" name="triadmissiondate">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>410</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>tri admission date</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
